json_journal/charts.py

```diff
@@ -1,7 +1,12 @@
+'''Create pie chart'''
```

Comment:
Is the purpose of this file only a pie chart? Or potentially broader in future?

Reply:
done - also changed a couple of others based on this comment to be broader